### PR TITLE
[stable/seq] Added seq

### PR DESCRIPTION
**Pull request by gertjvr**
**What this PR does / why we need it**:
This PR helps setup logging infrastructure, Seq creates the visibility you need to quickly identify and diagnose problems in complex applications and microservices. Empower your team to build better software by centralizing, searching, and alerting on structured application logs. 

**Special notes for your reviewer**:
To access the ui you need to run `kubectl port-forward {POD_NAME} 5341:80`

### CODE REVIEW

File: incubator/seq/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: seq
+version: 0.1.0
+appVersion: 5.0.832-pre
+description: Seq is the easiest way for development teams to capture, search and visualize structured log events! This page will walk you through the very quick setup process.
```
Comment (unguiculus) [inline incubator/seq/Chart.yaml]: The last sentence of the description probably doesn't make sense here.

File: incubator/seq/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+$ helm install seq
+```
+
+## Introduction
+
+This chart bootstraps a [Seq](https://hub.docker.com/r/datalust/seq/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (unguiculus) [inline incubator/seq/README.md]: We officially support the current and previous minor versions of Kubernetes. The chart won't work on 1.4 anyways.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a few minor things.

File: incubator/seq/Chart.yaml (hunk 1 above)
Comment (gertjvr) [reply incubator/seq/Chart.yaml]: Fixed

File: incubator/seq/README.md (hunk 2 above)
Comment (gertjvr) [reply incubator/seq/README.md]: Fixed

**Comment by unguiculus** [comment]
I'd suggest you move the PR to stable directly.

File: incubator/seq/values.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Default values for seq.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: datalust/seq
+  tag: latest
```
Comment (unguiculus) [inline incubator/seq/values.yaml]: Please use a release version.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/seq/values.yaml (hunk 3 above)
Comment (gertjvr) [reply incubator/seq/values.yaml]: Done

**Comment by gertjvr** [comment]
@unguiculus thank you for looking at the PR have made the changes as you suggested.

File: stable/seq/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+description: Seq is the easiest way for development teams to capture, search and visualize structured log events! This page will walk you through the very quick setup process.
+keywords:
+- seq
+- structured
+- logging
+home: http://getset.net/
+icon: https://avatars1.githubusercontent.com/u/5898109?s=200&v=4
+sources:
+- https://github.com/datalust/seq-tickets
+maintainers:
```
Comment (unguiculus) [inline stable/seq/Chart.yaml:13]: Any reason why you don't put the other owners here?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/seq/Chart.yaml (hunk 4 above)
Comment (gertjvr) [reply stable/seq/Chart.yaml:13]: I would be the one supporting this they maintain the seq images.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**